app/workers/evss/disability_compensation_form/submit_form526.rb

```diff
@@ -15,12 +17,12 @@ class SubmitForm526
 
       # This callback cannot be tested due to the limitations of `Sidekiq::Testing.fake!`
       sidekiq_retries_exhausted do |msg, _ex|
-        transaction_class.update_transaction(msg['jid'], :exhausted)
+        TRANSACTION_CLASS.update_transaction(msg['jid'], :exhausted)
         log_message_to_sentry(
```

Comment:
I think you said you tested this and it was making it all the way through to the metric call, but just double check that `log_message_to_sentry` is available in the callback. It's included (rather than extended) in the JobStatus concern which makes me think it'll be an instance method.

Reply:
I will double check this to verify